<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EepromLSC</class>
 <widget class="QWidget" name="EepromLSC">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(114, 159, 207);</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>30</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 25 28pt &quot;Microsoft YaHei UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>eeprom LSC</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>341</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>endianness</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_endianness"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>rGainMSB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_rGainMSB"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>grGainMSB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_grGainMSB"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>gbGainMSB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_gbGainMSB"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>bGainMSB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_bGainMSB"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>rIncrement</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_rIncrement"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>gbIncrement</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_gbIncrement"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>rolloffsize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_rolloffsize"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>rGainLSB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_rGainLSB"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>grGainLSB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_grGainLSB"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>gbGainLSB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_gbGainLSB"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>bGainLSB</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_bGainLSB"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>grIncrement</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_grIncrement"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,1">
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>bIncrement</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_bIncrement"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
